src/charts/helpers/axis.js

```diff
@@ -112,8 +124,32 @@ define(function(require) {
         };
     };
 
+    /**
+     * Returns tick object to be used when building the x axis
+     * @param {dataSorted} dataSorted       Chart data ordered by Date
+     * @param {Number} width                Chart width
+     * @param {String} [settings=null]      Optional forced settings for axis
+     * @return {object} tick settings for minor axis
+     */
+    const getSortedNumberAxis = (dataSorted, width) => {
+        const firstEntry = dataSorted[0].date;
+        const lastEntry = dataSorted[dataSorted.length - 1].date;
+        const timeSpan = lastEntry - firstEntry;
+
+        const minorTickValue = getMaxNumOfHorizontalTicksForNumberRanges(
+            width,
+            timeSpan
+        );
+
+        return {
+            tick: minorTickValue,
+            format: null
```

Comment:
Is `format:null` necessary? As I understand, by default it will be `undefined`. Would `null` be a better option? Just curious why.

Reply:
Oh no, just wanted to stick to the interface and be consistent. So I remove it completely, as it does not make sense.